### ISSUE TITLE
'in' queries seem to iterate over each and every element.

### ISSUE THREAD

**Issue by RobiFerentz** [balderdashy/sails-mongo#199]
When querying like so Model.find({id:[1,2,3,4,5,6,7]} , the adapter creates a different query for each id in the given array, instead of using the native mongo $in operator to get just one query.

**Comment by sailsbot** [comment]
Thanks for posting, @RobiFerentz.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
